includes/class-wc-google-gtag-js.php

```diff
@@ -233,36 +341,42 @@ protected function add_item( $order, $item ) {
 		$_product = $item->get_product();
 		$variant  = self::product_get_variant_line( $_product );
 
-		$code  = '{';
-		$code .= "'id': '" . self::get_product_identifier( $_product ) . "',";
-		$code .= "'name': '" . esc_js( $item['name'] ) . "',";
-		$code .= "'category': " . self::product_get_category_line( $_product );
+		$event_item = array(
+			'id'       => self::get_product_identifier( $_product ),
```

Comment:
I know it wasn't introduced in this PR, but we don't really need to use an underscore prefix especially since it will remain within the scope of this function. So I'd stick with just `$product`.

Reply:
Very true! I've [updated the variable name](https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/5675be7a712ee0912e5138476c87e8bcea86b165/includes/class-wc-google-gtag-js.php#L342) to be more in keeping with how things are done elsewhere.